client/src/App.js

```diff
@@ -18,6 +19,7 @@ function App () {
         <Route exact path='/' component={NavBar} />
```

Comment:
Okey this make the first page at app is Error 500 because you render navbar nothing else, you should render Events or rooms as default

Reply:
Events component not called at all here!

Reply:
Events component doesn't appear any way, even if I put it instead of NavBar in the App.js